src/cli/commands/init.js

```diff
@@ -31,6 +31,8 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
     email: config.getOption('init-author-email'),
     url: config.getOption('init-author-url'),
   };
+  const _defaultPrivateFlag = String(config.getOption('init-private'));
+  const defaultPrivateFlag = _defaultPrivateFlag === 'undefined' ? '' : _defaultPrivateFlag;
```

Comment:
Why not simply
```
const defaultPrivateFlag = config.getOption('init-private') || '';
```

This is actually safer since it allows the literal `'undefined'` value too. (not that I expect it to be used but it is "more accurate")

Reply:
Hi. I tried this way first. But this not pass the lint test. :(

Reply:
You can see here: https://circleci.com/gh/yarnpkg/yarn/tree/pull%2F4377
Ok, I just thought of a shorter way. Let me try.